docs/app/interfaces/ISection.ts

```diff
@@ -4,6 +4,7 @@ export interface ISection {
     component: string;
     codepen: boolean;
     version: 'AngularJS' | 'Angular';
+    hybrid: boolean;
```

Comment:
Possibly make this an optional property as it doesn't seem to be used in every section.